pkg/template/api/types.go

```diff
@@ -13,6 +13,9 @@ type Template struct {
 	unversioned.TypeMeta
 	kapi.ObjectMeta
 
+	// Optional: An instructional message for the user processing this template (escaped parameters will be substitued).
```

Comment:
how about: "An instructional message for the user that will be displayed when the template is instantiated (parameter substitution will be performed)."

(technically it is true that the message is handled during processing, but the only clients that are going to display the message are clients that are doing actual instantiation.  If the user just runs "oc process" on the template, the message will not be explicitly displayed).